app/Community/Listeners/NotifyMessageThreadParticipants.php

```diff
@@ -73,6 +74,8 @@ private function forwardToDiscord(
         MessageThread $messageThread,
         Message $message
     ): void {
+        $message->body = (new ReplaceUserShortcodesWithUsernamesAction())->execute($message->body);
```

Comment:
Shouldn't this just be reusing whatever thing is already doing this for recent forum posts? Then we'd get substitution for embedded achievements and games too. (Maybe it makes sense to ignore achievements for webhook so the user can get to the appropriate achievement. Are links a possibility?)

![image](https://github.com/user-attachments/assets/8ac98843-35b3-4fa6-8ce3-23f9fc127dcc)

Also, seems like the edit post function isn't converting the user link back to a name.

![image](https://github.com/user-attachments/assets/10f48e44-6326-40ba-b6d9-32970f82a850)

Reply:
Took care of both in latest.

The new action is still retained for the Edit post form, but it's isolated only to that component now. Elsewhere, we reuse the `Shortcode` helper method.